.release/buildspec_stage.yml

```diff
@@ -22,7 +22,7 @@ phases:
       - |
         for artifact in `cat ./$GIT_COMMIT_ID.manifest`
         do
-            md5FileName="${artifact%.exe}"
```

Comment:
im so confused - we used to have ".exe" at the end of all md5 files? but that's clearly not the case here: https://github.com/aws/copilot-cli/releases what am I misunderstanding?

Reply:
so for some reason on the windows file, we trim off the `.exe`, but only for the md5. the `.asc` file we leave it on. i just want them to be the same :sob:. 

so today our outputs are:
```
copilot-windows.exe
copilot-windows.exe.asc
copilot-windows.md5
```

after this change (theoretically):
```
copilot-windows.exe
copilot-windows.exe.asc
copilot-windows.exe.md5
```

Reply:
ok my bad! I didn't realize "%.exe" removed ".exe" 😂 now evreything makes sense

Reply:
ok my bad! I didn't realize "%.exe" removed ".exe" 😂 now evreything makes sense